### PR TITLE
set the viewport size as the same of the camera size

### PR DESCRIPTION
**Pull request by gustavoneves12**
depends on: https://github.com/Brazilian-Institute-of-Robotics/gui-orogen-vizkit3d_world/pull/5

### CODE REVIEW

File: tasks/Task.cpp [hunk 1]
```diff
@@ -25,6 +25,17 @@ Task::~Task()
...
 {
+    /**
+     * Apply the camera parameters
+     */
+    underwater_camera_simulation::CameraParams params = _camera_params.get();
+
+    /**
+     * set the viewport size the scene size of the camera
+     */
+    _viewport_width.set((params.width <= 0) ? 800 : params.width);
```
Comment (doudou) [inline tasks/Task.cpp]: `<= 0` ???? Don't overdo things, return an error if the width/height is not valid, there is no "natural" resolution for this component (unlike a camera which has a native resolution ... I guess)

**Comment by dbcesar** [comment]
Done!

File: tasks/Task.cpp [hunk 2]
```diff
@@ -25,6 +25,25 @@ Task::~Task()
...
+    /**
+     * set the viewport size the scene size of the camera
+     */
+     if (params.width <=0 || params.height <=0)
+     {
+         std::invalid_argument("The camera width and height must be greater than zero");
+     }
+     else
+     {
+         _width.set(params.width);
```
Comment (doudou) [inline tasks/Task.cpp]: Oh this is bad ... same configuration parameter in two different places ... Hugh.

What is the point of the width/height in CameraParams if it does not set anything ? My understanding is that the actual width/height is really set by the width and height properties, so I would just remove it from the params structure.

**Comment by doudou** [comment]
Ping @dbcesar

**Comment by doudou** [comment]
Or is it @gustavoneves12 ?

**Comment by gustavoneves12** [comment]
I am going to change it!!!

**Comment by gustavoneves12** [comment]
@doudou, Diego is going to remove the width/height from camera params.

**Comment by doudou** [comment]
:+1:

**Comment by dbcesar** [comment]
@doudou and @gustavoneves12. I've tested and it is working fine. From my side it can already be merged.

**Comment by dbcesar** [comment]
I'm merging it.

**Pull request merged**